dash_cytoscape/CyLeaflet.py

```diff
@@ -0,0 +1,118 @@
+from dash import (
+    clientside_callback,
+    ClientsideFunction,
+    Output,
+    Input,
+    html,
+    dcc,
+)
+
+import dash_cytoscape as cyto
+import dash_leaflet as dl
+
+
+class CyLeaflet(html.Div):
+    def __init__(self, id, cytoscape_props=None, leaflet_props=None):
+        cytoscape_props = cytoscape_props or {}
+        leaflet_props = leaflet_props or {}
+        elements = cytoscape_props["elements"]
```

Comment:
I think I found a bug here. 

If a user instantiates a CyLeaflet object and omits the prop `cytoscape_props`, ie:
```
cytoscape_props=dict(
    elements=[],
    stylesheet=cy_stylesheet,
)
```
This will throw an error with line 18, with an invalid elements key. I think the solution would be to either have line 16 as:
`cytoscape_props = cytoscape_props or {"elements": []}`
or make `cytoscape_props` a required prop.